tests/instance/assess_instance_runs.py

```diff
@@ -1,25 +1,26 @@
+import datetime
+
 import flywheel
 import pandas as pd
 
+versions = ["1.0.0_1.0.20200331", "1.1.0_1.0.20201102_dev0"]
+
 
 input_dir = "~/Documents/flywheel/gears/dcm2niix/tests/instance"
```

Comment:
This makes this test dependent on a valid user's home directory and the existing subdirectories.

I don't see existence tested or creation enforced anywhere.

Not a blocker..... thoughts toward when we have minicore to test on.

Reply:
Again, we do not yet have CI/CD (automation) of end-to-end testing of Gears. These instance tests are for my testing of the Gear in place of that automation and are not meant to be "production-ready". I understand that other Gear authors may not do it this way and so that might be confusing. Think of it as a way to automate end-to-end testing in an "as best we can" sort of way right now without actually needing to click buttons in the UI.